tests/abe/cover_crypt.test.ts

```diff
@@ -15,7 +15,7 @@ const MEDIUM_SECRET_MKG_USK_ACCESS_POLICY =
 // Plaintext example is: My secret message
```

Comment:
rename folder `tests/abe` in `tests/cover_crypt`?

Reply:
Yes will do another PR on changing folder structure since it's pretty complicated right now for no reason…